src/ui/buttons/buttonDesigns/divideLogoTopTextBottomAnimation.jsx

```diff
@@ -0,0 +1,163 @@
+/* @flow */
+/** @jsx node */
+import { LOGO_CLASS } from '@paypal/sdk-logos/src';
+import { node, Fragment, type ChildType } from 'jsx-pragmatic/src';
+
+import { CLASS } from '../../../constants';
+import { BUTTON_SIZE_STYLE } from '../config';
+
+import type {  ContentOptions } from './types';
+
+type AnimationProps = {|
+    paypalButtonContainerElement : Object,
+    animationConfig : Object,
+    designContainer : Object
+|};
+
+export const LOGO_TOP_TEXT_BOTTOM_CONFIG = {
```

Comment:
hey @lucasfrotam , you can create a constants file and import this constant in here, we already did that in a previous pr: https://github.com/paypal/paypal-checkout-components/pull/1807/files#diff-8c1e1f81a9e134254530315fef6dc4fe05cfe18f95572bb96771b86c65c14c40R8, the idea is to use the same constant for all the animations we can

Reply:
done